src/pathfinding_service/api.py

```diff
@@ -225,30 +234,44 @@ def get(self, token_network_address: TokenNetworkAddress) -> Tuple[dict, int]:
 
 
 class InfoResource(PathfinderResource):
-    version = pkg_resources.get_distribution('raiden-services').version
+    version = pkg_resources.get_distribution("raiden-services").version
 
     def get(self) -> Tuple[dict, int]:
         price = 0
-        settings = 'PLACEHOLDER FOR PATHFINDER SETTINGS'
-        operator = 'PLACEHOLDER FOR PATHFINDER OPERATOR'
-        message = 'PLACEHOLDER FOR ADDITIONAL MESSAGE BY THE PFS'
+        settings = "PLACEHOLDER FOR PATHFINDER SETTINGS"
+        operator = "PLACEHOLDER FOR PATHFINDER OPERATOR"
+        message = "PLACEHOLDER FOR ADDITIONAL MESSAGE BY THE PFS"
 
         return (
             {
-                'price_info': price,
-                'network_info': {
-                    'chain_id': self.pathfinding_service.chain_id,
-                    'registry_address': self.pathfinding_service.registry_address,
+                "price_info": price,
+                "network_info": {
+                    "chain_id": self.pathfinding_service.chain_id,
+                    "registry_address": self.pathfinding_service.registry_address,
                 },
-                'settings': settings,
-                'version': self.version,
-                'operator': operator,
-                'message': message,
+                "settings": settings,
+                "version": self.version,
+                "operator": operator,
+                "message": message,
             },
             200,
         )
 
 
+class DebugEndpoint(PathfinderResource):
+    def get(  # pylint: disable=no-self-use
+        self, token_network_address: TokenNetworkAddress, source_address: Address
+    ) -> Tuple[dict, int]:
+        request_count = 0
```

Comment:
The idea of the `request_count` in the SP was to be able to differentiate between failed and successful requests. In this implementation it's basically just `len(responses) - 1`.

Reply:
hmm the failed request would now count as a request with set to `"responses": [ ]`. So the count would not include requests that did not reach the pfs. What do you suggest?

Reply:
so whenever the pfs throws `NetworkXNoPath` or `NodeNotFound` the debug endpoint adds a request and an empty list [ ] to the request deque. I am not sure how we could track failed requests on the pfs side apart from that

Reply:
we take the `request_count` for now as it is and improve later :)